.decent_ci-Linux.yaml

```diff
@@ -1,13 +1,14 @@
 compilers:
   - name: "gcc"
-    version: "11.4"
+    version: "13.2"
     cmake_extra_flags: -DLINK_WITH_PYTHON:BOOL=ON -DPYTHON_CLI:BOOL=OFF -DPython_REQUIRED_VERSION:STRING=3.12.2 -DPython_ROOT_DIR:PATH=~/.pyenv/versions/3.12.2/ -DBUILD_FORTRAN:BOOL=ON -DBUILD_TESTING:BOOL=ON -DENABLE_GTEST_DEBUG_MODE:BOOL=OFF -DBUILD_PERFORMANCE_TESTS:BOOL=ON -DVALGRIND_ANALYZE_PERFORMANCE_TESTS:BOOL=ON -DENABLE_PCH:BOOL=OFF
     collect_performance_results: true
     skip_regression: true
     s3_upload_bucket: energyplus
+    num_parallel_builds: 18
```

Comment:
The CI machine has 20 cores, but only 16 GB of RAM.  It was swapping, or getting close to swapping, so I explicitly pulled the number down here just a little.